Include/objimpl.h

```diff
@@ -251,9 +251,9 @@ PyAPI_FUNC(PyVarObject *) _PyObject_GC_Resize(PyVarObject *, Py_ssize_t);
 #ifndef Py_LIMITED_API
 typedef union _gc_head {
     struct {
-        union _gc_head *gc_next;
-        union _gc_head *gc_prev;
-        Py_ssize_t gc_refs;
+        union _gc_head *gc_next;  // NULL means the object is not tracked
+        uintptr_t gc_prev;  // Pointer to previous object in the list.
```

Comment:
When I read gc_prev, I expect it to only contain a pointer to the previous entry, but it contains two information. Would it be possible to find a better name? (gc_prev might be fine) Maybe "gc_prev_flags"?

Maybe add "_" prefix to "explain" that the field must be accessed through macros? (ex: _gc_prev_flags)

If you rename the field, it makes sure that applications accessing (read or... write!) directly gc_prev will break which is a good thing IMHO.

Can you also please document which flags are stored in this field? Finalized, collecting and tentatively unreachable. Mention maybe that the two last are only used internally and so are not accessible?

Reply:
> Can you also please document which flags are stored in this field? Finalized, collecting and tentatively unreachable. Mention maybe that the two last are only used internally and so are not accessible?

Finalized is documented below.  And I don't want document other two bits in this file
because they are tightly coupled with GC algorithm.  I added document in gcmodule.c instead.

Reply:
I changed member name to _gc_prev and _gc_next.
I don't want 3rd party touches them directly.